patches/server/0230-Implement-Expanded-ArmorStand-API.patch

```diff
@@ -87,6 +90,74 @@ index 82b9ee993b0d2e7e0685231f7bad2b85756ec959..f4065938bbfd04519d1363ee8781c316
 +    public boolean isSlotDisabled(org.bukkit.inventory.EquipmentSlot slot) {
 +        return getHandle().isDisabled(org.bukkit.craftbukkit.CraftEquipmentSlot.getNMS(slot));
 +    }
++
++    @Override
++    public org.bukkit.util.Vector getBodyRotation() {
++        return fromNMSRotation(getHandle().bodyPose);
++    }
++
++    @Override
++    public void setBodyRotation(org.bukkit.util.Vector rotation) {
++        getHandle().setBodyPose(toNMSRotation(rotation));
++    }
++
++    @Override
++    public org.bukkit.util.Vector getLeftArmRotation() {
++        return fromNMSRotation(getHandle().leftArmPose);
++    }
++
++    @Override
++    public void setLeftArmRotation(org.bukkit.util.Vector rotation) {
++        getHandle().setLeftArmPose(toNMSRotation(rotation));
++    }
++
++    @Override
++    public org.bukkit.util.Vector getRightArmRotation() {
++        return fromNMSRotation(getHandle().rightArmPose);
++    }
++
++    @Override
++    public void setRightArmRotation(org.bukkit.util.Vector rotation) {
++        getHandle().setRightArmPose(toNMSRotation(rotation));
++    }
++
++    @Override
++    public org.bukkit.util.Vector getLeftLegRotation() {
++        return fromNMSRotation(getHandle().leftLegPose);
++    }
++
++    @Override
++    public void setLeftLegRotation(org.bukkit.util.Vector rotation) {
++        getHandle().setLeftLegPose(toNMSRotation(rotation));
++    }
++
++    @Override
++    public org.bukkit.util.Vector getRightLegRotation() {
++        return fromNMSRotation(getHandle().rightLegPose);
++    }
++
++    @Override
++    public void setRightLegRotation(org.bukkit.util.Vector rotation) {
++        getHandle().setRightLegPose(toNMSRotation(rotation));
++    }
++
++    @Override
++    public org.bukkit.util.Vector getHeadRotation() {
++        return fromNMSRotation(getHandle().headPose);
++    }
++
++    @Override
++    public void setHeadRotation(org.bukkit.util.Vector rotation) {
++        getHandle().setHeadPose(toNMSRotation(rotation));
++    }
++
++    private static org.bukkit.util.Vector fromNMSRotation(Rotations old) {
```

Comment:
~~I think these nms methods for converting the rotation should be moved from the api module to server such as with potion effects have methods in CraftPotionUtil that convert to and from nms types.~~
Disregard this is already in the server side lol